src/web_service.iced

```diff
@@ -35,6 +35,12 @@ class WebServiceBinding extends Base
 
   #---------------
 
+  _type_v2 : (revoke_flag) ->
+    if @revoke? or revoke_flag then constants.sig_types_v2.web_service_binding_with_revoke
```

Comment:
there's one if the chainlink is loaded, and one if we're doing checks over raw json before it's loaded.